website/src/pages/en/subgraphs/explorer.mdx

```diff
@@ -161,69 +181,77 @@ In the Epochs section, you can analyze on a per-epoch basis, metrics such as:
+> Subgraphs deployed with the CLI for testing purposes will not show up here. Subgraphs will only show up when they are published to the decentralized network.
 
 ![Explorer Image 11](/img/Subgraphs-Overview.png)
 
-### Indexing Tab
+#### Indexing Tab
 
-In the Indexing tab, you’ll find a table with all the active and historical allocations towards Subgraphs. You will also find charts where you can see and analyze your past performance as an Indexer.
+> If you haven't indexed, you see links to stake to index Subgraphs and browse Subgraphs on Graph Explorer.
 
-This section will also include details about your net Indexer rewards and net query fees. You’ll see the following metrics:
+The Indexing tab displays a table where you can review active and historical allocations to Subgraphs.
 
-- Delegated Stake - the stake from Delegators that can be allocated by you but cannot be slashed
-- Total Query Fees - the total fees that users have paid for queries served by you over time
-- Indexer Rewards - the total amount of Indexer rewards you have received, in GRT
-- Fee Cut - the % of query fee rebates that you will keep when you split with Delegators
-- Rewards Cut - the % of Indexer rewards that you will keep when splitting with Delegators
-- Owned - your deposited stake, which could be slashed for malicious or incorrect behavior
+Track your Indexer performance with visual charts and key metrics, including:
 
-![Explorer Image 12](/img/Indexer-Stats.png)
+- Delegated Stake - Stake from Delegators that can be allocated by you but cannot be slashed
+- Total Query Fees - Cumulative fees from served queries
+- Indexer Rewards (in GRT) - Total rewards earned
+- Fee Cut & Rewards Cut - the % of query fee rebates and Indexer rewards you'll keep when you split with Delegators
+- Owned Stake - your deposited stake, which could be slashed for malicious or incorrect behavior
 
-### Delegating Tab
+![Explorer Image 12](/img/Indexer-Stats.png)
 
-Delegators are important to the Graph Network. They must use their knowledge to choose an Indexer that will provide a healthy return on rewards.
+#### Delegating Tab
 
-In the Delegators tab, you can find the details of your active and historical delegations, along with the metrics of the Indexers that you delegated towards.
+> To learn more about the benefits of delegating, check out [delegating](/resources/roles/delegating/delegating/)
 
-In the first half of the page, you can see your delegation chart, as well as the rewards-only chart. To the left, you can see the KPIs that reflect your current delegation metrics.
+The Delegators tab displays your active and historical delegations, along with the metrics for the Indexers you've delegated to.
 
-The Delegator metrics you’ll see here in this tab include:
+Top Section:
 
+- View delegation and rewards-only charts
+- Track key metrics:
 - Total delegation rewards
-- Total unrealized rewards
-- Total realized rewards
+- Unrealized rewards
+- Realized Rewards
 
```

Comment:
Should these be indented (are they the "key metrics" mentioned in the previous bullet point)?